spec/serializers/api/v2/serializable_course_spec.rb

```diff
@@ -93,4 +93,30 @@
       it { expect(subject["attributes"]).to include("is_send?" => true) }
     end
   end
+
+  context "subjects & level" do
+    context 'with no subjects' do
+      let(:course) { create(:course, subject_count: 0) }
+      it { expect(subject["attributes"]).to include("level" => "secondary") }
```

Comment:
Is this the default we want?

Reply:
No idea, but previously it would of been a secondary welsh corpse.

There should be more subject matter work, but current aim is to make it work and refine later.

@benilovj to clarify

Reply:
checked in meatsphere, its fine